stock_split_picking/models/stock_picking.py

```diff
@@ -24,6 +24,19 @@ def _check_split_process(self):
                 )
             )
 
+        if all(
+            [
+                x.qty_done == x.product_uom_qty
+                for x in self.move_line_ids.filtered(lambda m: m.qty_done != 0.0)
```

Comment:
same remark here but for `float_is_zero` https://github.com/odoo/odoo/blob/14.0/odoo/tools/float_utils.py#L102

Reply:
not blocking for this 😄